### PR TITLE
Update audit-log-activities.md with Microsoft Defender Experts

### PR DESCRIPTION
**Pull request by hyukjinkwonMicrosoft**
Added new section for Microsoft Defender Experts activities

### CODE REVIEW

File: microsoft-365/compliance/audit-log-activities.md [hunk 1]
```diff
@@ -314,6 +314,14 @@ The following table lists the activities in information barriers that are logged
 | Changed segments of a site | SegmentsChanged | A SharePoint or global administrator changed one or more information barriers segments for a site. |
 | Removed segments from a site | SegmentsRemoved | A SharePoint or global administrator removed one or more information barriers segments from a site. |
 
+## Microsoft Defender Experts activities
+
+The following table lists the activities in Microsoft Defender Experts that are logged in the Microsoft 365 audit log.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/audit-log-activities.md]:
```suggestion
The following table lists the activities in Microsoft Defender Experts that are signed into the Microsoft 365 audit log.
```

File: microsoft-365/compliance/audit-log-activities.md [hunk 2]
```diff
@@ -314,6 +314,14 @@ The following table lists the activities in information barriers that are logged
...
+|Friendly name|Operation|Description|
+|:----------------|:------------|:--------------|
+| Defender Experts Analyst Permission Created | DefenderExpertsAnalystPermissionCreated | An administrator granted one or more role permissions to Defender Experts analysts to investigate incidents or remediate threats.|
+| Defender Experts Analyst Permission Modified | DefenderExpertsAnalystPermissionModified | An administrator modified role permissions to Defender Experts analysts to investigate incidents or remediate threats.|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/audit-log-activities.md]:
```suggestion
| Defender Experts analyst permission created | DefenderExpertsAnalystPermissionCreated | An administrator granted one or more role permissions to Defender Experts analysts to investigate incidents or remediate threats.|
| Defender Experts analyst permission modified | DefenderExpertsAnalystPermissionModified | An administrator modified role permissions to Defender Experts analysts to investigate incidents or remediate threats.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyukjinkwonMicrosoft, kindly commit the suggested changes. Many thanks.

**Comment by hyukjinkwonMicrosoft** [comment]
Hi @JohanFreelancer9 I've committed the changes you suggested. Thank you!

Since I don't have write access to this repo, I can't merge my PR. Could you assist me with this?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by robmazz** [comment]
@yogkumgit  - FYI, discussed with PM, as internal stakeholder - he is opening the content request in our ADO per our process

**Pull request closed without merging**